setup.py

```diff
@@ -42,7 +42,7 @@
     license="Apache 2",
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
-    install_requires=["pytket ~= 1.11"],
+    install_requires=["pytket ~= 1.11", "kahypar ~= 1.3.5"],
```

Comment:
Just wondering: did we not include `cuquantum-python` here for some particular reason?

Reply:
So that one can install package on a machine without GPU... Not sure if it is sensible though!